statistics/update_test.go

```diff
@@ -223,3 +223,50 @@ func (s *testStatsUpdateSuite) TestTxnWithFailure(c *C) {
 	stats1 = h.GetTableStats(tableInfo1.ID)
 	c.Assert(stats1.Count, Equals, int64(rowCount1+1))
 }
+
+func (s *testStatsUpdateSuite) TestAutoUpdate(c *C) {
+	store, do, err := newStoreWithBootstrap()
```

Comment:
Can we use a unified store in testStatsUpdateSuite? Bootstrap make the test run a long time.